packages/pricing/src/models/rule-type.ts

```diff
@@ -0,0 +1,49 @@
+import { generateEntityId } from "@medusajs/utils"
+import { PricingTypes } from "@medusajs/types"
+import {
+  BeforeCreate,
+  Entity,
+  Enum,
+  ManyToOne,
+  OptionalProps,
+  PrimaryKey,
+  Property,
+} from "@mikro-orm/core"
+
+type OptionalFields =
+  | "default_priority"
+  | "kind"
+  | "is_dynamic"
+  
+@Entity()
+class RuleType {
+  [OptionalProps]?: OptionalFields 
+
+  @PrimaryKey({ columnType: "text" })
+  id!: string
+
+  @Property({ columnType: "text" })
+  name: string
+
+  @Property({ columnType: "text" })
+  key_value: string
+
+  @Property({ columnType: "integer", default: 0 })
+  default_priority: number
+  
+  @Enum({
+    items: () => PricingTypes.RuleTypeKind,
+    default: PricingTypes.RuleTypeKind.FILTER,
+  })
+  kind: PricingTypes.RuleTypeKind
+  
+  @Property({ columnType: "boolean", default: false })
+  is_dynamic: boolean
```

Comment:
if we are adding this property right now, it would make sense to also add the "evaluate_expression" that will contain what will be evaluated at run-time.
At this first iteration I suggest to remove it.